spec/features/thredded/user_follows_topic_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+feature 'Logged in user' do
+  before { user.log_in }
+
+  scenario 'can follow a topic' do
+    an_unfollowed_topic.visit_topic
+    expect(page).to have_button('Follow')
+    click_on 'Follow'
+    expect(page).to_not have_button('Follow')
+  end
+
+  scenario 'can unfollow a topic' do
+    a_followed_topic.visit_topic
+    expect(page).to have_button('Stop following')
+    click_on 'Stop following'
+    expect(page).to_not have_button('Stop following')
+  end
+
+  scenario 'can see follow status in list of topics'
+
+  let(:user) { PageObject::User.new(create(:user)) }
+  let(:an_unfollowed_topic) do
+    topic = create(:topic, with_posts: 1, messageboard: create(:messageboard))
+    PageObject::Topic.new(topic)
+  end
+  let(:a_followed_topic) do
+    topic = create(:topic, with_posts: 1, messageboard: create(:messageboard))
+    Thredded::UserTopicFollow.create_unique(user.user.id, topic.id)
+    PageObject::Topic.new(topic)
+  end
```

Comment:
Would you be ok with putting these `let` at the top of the feature? I'm a fan of the [xunit 4-phase style](http://xunitpatterns.com/Four%20Phase%20Test.html) where it's read 

1. setup
2. exercise
3. verify
4. tear-down

If this test grows much larger and these let's get pushed down to the bottom of the file/buffer it *might* end up with a bit of a [mystery guest](https://robots.thoughtbot.com/mystery-guest) smell.